### PR TITLE
chore: Fix test summary to depend on all other tasks

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: .github/workflows/validation.yml [hunk 1]
```diff
@@ -282,6 +282,10 @@ jobs:
     if: ${{ always() }}
     needs:
       - test-it
+      - test-java
+      - test-typescript
+      - test-it
```
Comment (cromoteca) [inline .github/workflows/validation.yml:287]: This is duplicate

Comment (Artur-) [reply .github/workflows/validation.yml:287]: True that
True that

**Comment by Artur-** [comment]
Let's check that it looks correct in #1434

**Comment by Artur-** [comment]
looks correct to me 
image

**Review by manolo — APPROVED** [review]

**Pull request merged**